frigate/storage.py

```diff
@@ -86,26 +86,48 @@ def calculate_camera_usages(self) -> dict[str, dict]:
 
         return usages
 
+    def calculate_storage_recovery_target(self) -> int:
```

Comment:
I have concerns about this being a per-camera config because in reality it does not behave that way. This logic aggregates the per-camera config to a total number, but in reality storage cleanup is just being run and deleting the oldest recordings regardless of which camera they belong to